### DISCUSSION TITLE
Update existing dependency dashboard when `mode=silent`

### DISCUSSION THREAD

**Discussion by sisp** [renovatebot/renovate#29848] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

I'm trying to use [`mode=silent`](https://docs.renovatebot.com/configuration-options/#mode) for triggering a rebase-only Renovate run. It works as expected except that when the dependency dashboard is enabled and I tick a checkbox there to request a rebase of a MR, Renovate won't reset the checkbox afterwards. I'm aware of the following statement in the documentation:

> When in silent mode, Renovate does not create issues (such as Dependency Dashboard, or due to config errors) or Config Migration PRs, even if enabled.

However, I wonder whether it would be reasonable to _update an existing_ dependency dashboard issue. WDYT?

**Comment by viceice** [comment]
not a bug

**Reply by sisp** [reply]
What's the rationale for not updating an _existing_ dependency dashboard?

**Reply by rarkins** [reply]
A bug is something which is not behaving as designed/documented. You're asking for a change of design.

**Comment by fgreinacher** [comment]
_(Disclaimer: @sisp is a colleague of mine and I directed him towards the `mode` option 😄 )_

This would be super helpful especially in cases where you want to run Renovate on a relaxed schedule, but still want a way to get rebase requests processed quickly/out-of-band.

I would also have expected the `silent` mode to work like this, but of course this could also be a new mode.

**Reply by rarkins** [reply]
I'm not opposed to the suggestion, but I'm opposed to people who file invalid bugs to Open Source projects. It's usually people trying to "twist" the description into a bug to make it seem more important, and more often than not it's people from large companies. It wastes our time.

**Reply by sisp** [reply]
Apologies if you perceived this as "trying to twist the description into a bug" to raise priority, as this was not the intention. IMHO, it's a matter of perspective whether this might be a bug or a design change. The docs state that "Renovate does not _create_ issues (such as Dependency Dashboard [...]", but there is no statement about the (intended) behavior for existing dependency dashboards. With `mode=silent`, Renovate still honors a ticked checkbox in an existing dependency dashboard, so not updating the _existing_ dependency dashboard (i.e., unticking the checkbox again) feels like an oversight (or a light bug) either in the implementation or in the design.

Let's not get hung up on semantics please. :pray:

**Reply by rarkins** [reply]
Corporate users telling open source maintainers to "not get hung up on semantics" when they've explained that you're wasting their time is not welcome in this repo either.

**Comment by rarkins** [comment, marked as answer]
Rejected

---

Edit by @HonkingGoose, I'm improving the docs with:

- #29866
